docs/source/models/cause.rst

```diff
@@ -4,17 +4,119 @@
 Modeling Causes
 ===============
 
-.. todo::
-
-   Overview of modeling GBD causes.
+What is included in this chapter: models of causes that are represented as
+finite state Markov chain (this chapter will also explain what a finite state
+Markov chain is!)
+
+There are models of things that you might consider causes which are outside of
+the scope of this chapter, because they do not fit into the Markov approach.
+This chapter does not include “PAF of 1” causes, like protein-energy
+malnutrition (PEM).  It also does not include things that are sometimes
+considered a disease but are deemed “risk factors” by GBD, such as obesity.
+Unsafe water would not fit into the cause model paradigm either---it is a cause
+of diarrheal disease, but it is a risk factor in the GBD taxonomy.  Latent
+tuberculosis infection (LTBI) does fit into this chapter, but just barely.
 
 .. contents:
+   :local:
+
+
+What is a "cause"?
```

Comment:
I think having a combined What is a cause/what is a cause model section might be more helpful.  A lot of the work in producing a good cause model is bridging that gap, so a compare/contrast might be helpful.  It also seems like there are three levels to this:

- The cause in the world (ie the thing we want to model)
- The GBD model(s) of the cause (fatal, nonfatal, intermediate things like dismod models).
- The **cause model**, ie the vivarium model of the cause.